pulp_ansible/app/tasks/collections.py

```diff
@@ -280,6 +282,20 @@ def create_collection_from_importer(importer_result):
     return collection_version
 
 
+def _get_backend_storage_url(artifact_file):
+    """Get artifact url from pulp backend storage."""
+    if settings.DEFAULT_FILE_STORAGE == "pulpcore.app.models.storage.FileSystem":
+        url = None
+    elif settings.DEFAULT_FILE_STORAGE == "storages.backends.s3boto3.S3Boto3Storage":
+        parameters = {"ResponseContentDisposition": "attachment;filename=archive.tar.gz"}
```

Comment:
uses `"archive.tar.gz"` rather than `content_artifact.relative_path` since this only has artifact file from PulpTemporaryFile

Reply:
generic name `"archive.tar.gz"` made me think about debugging/logging, maybe `f"{collection_name}.tar.gz"`